releasedir/interfaces.go

```diff
@@ -73,6 +73,8 @@ type BlobsDir interface {
 
 	TrackBlob(string, io.ReadCloser) (Blob, error)
 	UntrackBlob(string) error
+
+	ContainsSymlinks() bool
```

Comment:
may be instead of exposing it as a public method, it makes more sense to call it from inside SyncBlobs()? /cc @dpb587